### DISCUSSION TITLE
Texture/Material override for custom spraypaints

### DISCUSSION THREAD

**Discussion by GazziFX** [SmartlyDressedGames/Unturned-3.x-Community#4521] [Ideas]
I think it would be really cool if you can spray vehicles with some textures like military camo

**Comment by grewqcxeqeq** [comment]
I think this is the best thing that Nelson could do, I play single player and I miss repainting and adding my custom colors and I wanted Nelson to add repainting weapons and return them so that I could play single player with friends and there was no need install the server............

**Comment by postmvn** [comment]
Yes, I thought vehicle colors would be made in that way from the start. With material palettes for vehicles it would be possible to do different combinations for paints and not just single color (like police cars that have blue white or black white combos etc). I hope it would be at least added later on.

**Comment by DanielWillett** [comment]
This would be cool, could definitely find a use for this. Would also be nice if redirect assets included this option like they do paint colors.

**Comment by grewqcxeqeq** [comment]
@SDGNelson

**Comment by Jdance-Media** [comment]
Material overrides was specifically mentioned and somewhat addressed on the forums. However, I want to stem from this discussion by saying that it would be better to have a basic system in place to override the materials contained within any asset, rather than having paint assets specific only to vehicles.
